.ci/windows-build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o errexit
+set -x
+
+CFLAGS_FOR_OVS="-g -O2"
+EXTRA_OPTS="--with-pthread=`realpath ./PTHREADS-BUILT | xargs cygpath -m`"
+
+function configure_ovs()
+{
+    ./boot.sh
+    ./configure CC="./build-aux/cccl" LD="`which link`" \
+    LIBS="-lws2_32 -lShlwapi -liphlpapi -lwbemuuid -lole32 -loleaut32" \
+    CFLAGS="${CFLAGS_FOR_OVS}" $* || { cat config.log; exit 1; }
+}
+
+
+OPTS="${EXTRA_OPTS} ${OPTS} $*"
+configure_ovs $OPTS
+make -j4 || { cat config.log; exit 1; }
```

Comment:
Please change to `make -j` so it will use as many threads as the container / VM has